.prettierrc.toml

```diff
@@ -6,5 +6,4 @@ quoteProps = 'consistent'
 trailingComma = 'all'
 arrowParens = 'always'
 proseWrap = 'preserve'
-tabWidth = 4
```

Comment:
We want to be using the definitions from `.editorconfig`, which are different between file types.